apps/preview/next/pages/showcases/OrderSummary/OrderSummary.tsx

```diff
@@ -14,6 +14,9 @@ const orderDetails = {
 export default function OrderSummary() {
   const [inputValue, setInputValue] = useState('');
   const [promoCode, setPromoCode] = useState(0);
+  const [RemovedCodeAlert, setRemovedCodeAlert] = useState(false);
+  const [AddedCodeAlert, setAddedCodeAlert] = useState(false);
+  const [errorAlert, setErrorAlert] = useState(false);
```

Comment:
I feel like it could be done using two states:
1. isOpen using `useDisclosure`
2. promo code submission result: `success | error | reset`

Then markup should look a bit different. We can have a call where I can explain a bit more.

Reply:
As we discussed we need separated to have possibility to show more than one alert at the same time